addon/services/data-pipeline.js

```diff
@@ -111,12 +110,16 @@ export default Service.extend({
   },
 
   _generateData(incoming, action) {
+    if (typeof document === 'undefined') {
+      return false;
```

Comment:
On this, do you want to not return ANYTHING, or are the other keys valid in Fastboot (e.g. `action`, `browser_id`, `client`) and you only can't use the `external_referrer`?

Reply:
Hmm. browser_id definitely isn't set in fastboot, but after a closer look, maybe this isn't the right place to bail out.